Shell/Preprocess.cpp

```diff
@@ -393,6 +393,9 @@ void Preprocess::preprocess(Problem& prb)
      twee.apply(prb,(env.options->tweeGoalTransformation() == Options::TweeGoalTransformation::GROUND));
    }
 
+   // refresh symbol usage counts, can skip unused symbols for equality proxy
+   prb.getProperty();
```

Comment:
How about we hide this below under the `if` so that we tax only `the ep!=off` branch?

Reply:
Good idea, fixed.

Reply:
Actually could even further optimise and do this only if `ep == RSTC`, because we only need to know for congruence axioms. Let me know if you care.

Reply:
I think this is fine, thank you!